lib/frontend/src/frontend.rs

```diff
@@ -231,8 +220,8 @@ impl<'a, Variable> FunctionBuilder<'a, Variable>
             func: func,
             builder: builder,
             position: Position {
-                ebb: Ebb::default(),
-                basic_block: Block::default(),
+                ebb: Ebb::new(0),
```

Comment:
Yeah I never read the value and I didn't want the hassle of an option type (lots of `unwrap()`).